lighty-core/dependency-versions/pom.xml

```diff
@@ -33,7 +33,7 @@
             <dependency>
                 <groupId>org.opendaylight.odlparent</groupId>
                 <artifactId>odlparent</artifactId>
-                <version>14.1.1</version>
```

Comment:
document odlparent version in commit msg

Reply:
resolved